.github/workflows/ci.yml

```diff
@@ -21,12 +21,37 @@ jobs:
       - name: Install dependencies
         run: forge install
 
-      - name: Check backward compatibility
-        run: |
-          forge build --skip test --use solc:0.8.0
-          forge build --skip test --use solc:0.7.6
-          forge build --skip test --use solc:0.7.0
-          forge build --skip test --use solc:0.6.2
+      - name: Measure compilation time of Test with 0.8.17 --via-ir
```

Comment:
Each of these takes about 40 seconds. Let's create a separate job after `fmt` instead?

Reply:
Not sure I understand the suggestion here, each job (build, test, fmt) runs in parallel, so this seems ok

Reply:
For example, see how test/fmt finished but build is still going for the last commit 
<img width="508" alt="image" src="https://user-images.githubusercontent.com/17163988/201385414-9029357b-b9c6-47d9-bb16-253dd022ee39.png">

Reply:
I forgot. Sorry for nit picking- can we move them after the backward compatibility check then? I’d like the workflow to fail fast, instead of waiting 3 minutes for those to finish, to then check backward compatibility, which is more likely to catch something.

Reply:
Makes sense. Done in fab939e, also added an 0.6.12 check for good measure